src/webgpu/shader/validation/decl/var_access_mode.spec.ts

```diff
@@ -130,7 +137,7 @@ g.test('explicit_access_mode')
         .combine('addressSpace', kNonHandleAddressSpaces)
         .expand('info', infoExpander)
         .combine('explicitSpace', [true, false])
-        .filter(t => compatibleAS(t))
+        .filter(t => varDeclCompatibleAddressSpace(t))
         .combine('explicitMode', [true])
         .combine('accessMode', keysOf(kAccessModeInfo))
         .combine('stage', ['compute' as ShaderStage]) // Only need to check compute shaders
```

Comment:
Github sucks, and I can't put a comment on L154, but I don't think `as readonly string[]` gives you anything. Can just be `t.params.info.accessModes.includes(t.params.accessMode)`